pkg/validation/providers.go

```diff
@@ -0,0 +1,67 @@
+package validation
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/oauth2-proxy/oauth2-proxy/v7/pkg/apis/options"
+)
+
+// validateMultipleProviders is the initial validation migration for multiple providrers
+// It currently includes only logic which can verify the providers one by one and does not break the valdation pipe
+func validateMultipleProviders(o *options.Options) []string {
```

Comment:
Do we need to call this `Multiple` providers or should we just call this whole thing `validateProviders`?